acmw-db-app/pages/api/account/[pid].js

```diff
@@ -20,6 +68,12 @@ export default async (req, res) => {
     
             if(pid === 'verify') {
                 result = await verify(body.email, body.password);
+            } else if(pid === 'generate-token') {
+                if(!body.email) throw ("Missing email address in request body.");
+                result = await generateToken(body.email);
+            } else if(pid === 'reset-pw') {
+                if(!body.password || !body.token) throw ("Missing token and/or new password");
+                result = await resetPassword(body.token, body.password);
```

Comment:
I'm confused why does this only need token and password instead of also needing email?